.github/workflows/_linux-test.yml

```diff
@@ -111,8 +111,17 @@ jobs:
         id: parse-ref
         run: .github/scripts/parse_ref.py
 
+      - name: Set Test step time
+        id: test-timeout
+        shell: bash
+        env:
+          JOB_TIMEOUT: ${{ inputs.timeout-minutes }}
+        run: |
+          echo "timeout=$((JOB_TIMEOUT-30))" >> "${GITHUB_OUTPUT}"
```

Comment:
Is 30 a rough upper boundary for how many minutes may have elapsed in this job before this step runs?